Rakefile

```diff
@@ -41,8 +41,9 @@ require 'rake/testtask'
 
 Rake::TestTask.new do |t|
   t.libs << 'test'
+  t.libs << 'lib'
```

Comment:
it doesn't seem to need it. in fact, I removed it earlier.  But, most rake test tasks had it so I decided to cargo cult.  I'm find either way. :shrug: